### PR TITLE
Add `radar` icon

### PR DESCRIPTION
**Pull request by danielbayley**

### CODE REVIEW

**Comment by karsa-mistmere** [comment]
While "radar" certainly comes across, I think this design may be a bit too dense, how about?
radar

**Comment by danielbayley** [comment]
@karsa-mistmere `push`ed…

File: icons/radar.svg [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="m13.4 10.6 5.7-5.7C17.3 3.1 14.8 2 12 2c-1.8 0-3.5.5-5 1.4" />
+  <path d="M2.3 9.6c-.2.8-.3 1.6-.3 2.4 0 5.5 4.5 10 10 10s10-4.5 10-10c0-1.3-.3-2.5-.7-3.6" />
+  <path d="M4 6h0" />
+  <path d="M16.2 7.8C15.2 6.7 13.7 6 12 6c-3.3 0-6 2.7-6 6 0 1.9.9 3.6 2.2 4.7" />
+  <path d="M15.8 16.7c1.4-1.1 2.2-2.8 2.2-4.7v-.3" />
+  <path d="M12 18h0" />
+  <circle cx="12" cy="12" r="2" />
```
Comment (jguddas) [inline icons/radar.svg:18]:
```suggestion
  <circle cx="12" cy="12" r="2" />
  <path d="M13.4 10.6L19.1 4.9" />
  <path d="M 19.1,4.9 A 10 10 0 0 0 7 3.4" />
  <path d="M 2.3,9.6 A 10 10 0 1 0 21.3 8.4" />
  <path d="M4 6L4.01 6" />
  <path d="M 16.2,7.8 A 6 6 0 1 0 8.2 16.7" />
  <path d="M 15.8,16.7 A 6 6 0 0 0 18 11.7" />
  <path d="M12 18L12.01 18" />
```

**Comment by ericfennis** [comment]
Can we try one alternative design the dot removed from the border, so we have only one dot?

**Comment by karsa-mistmere** [comment]
Various combinations of ideas so far:
image

**Comment by jguddas** [comment]
I prefer it with one dot on the outer and one dot on the inner ring.

**Comment by danielbayley** [comment]
> I prefer it with one dot on the outer and one dot on the inner ring.

The variation before the last commit? (Simple to revert)… I’m easy, on this one… Maybe put it to a vote?

**Comment by ericfennis** [comment]
Yes let's do a vote!
I vote for the right-bottom one.

**Comment by danielbayley** [comment]
> Yes let's do a vote! I vote for the right-bottom one.

Hmm, I think the middle rung is a bit too broken up on that one… So out of those I would probably go with top right.

But also have a couple of slight variations for consideration…

Untitled

**Comment by jguddas** [comment]
+1 for the top right one.

**Comment by danielbayley** [comment]
> Yes let's do a vote!

Maybe we could use something like a poll bot for this kind of thing…

https://github.com/marketplace/actions/github-polls-bot-action

**Comment by ericfennis** [comment]
@danielbayley Cool bot, will think of that one!

**Comment by ericfennis** [comment]
@karsa-mistmere You want do a final vote on this one?

**Comment by karsa-mistmere** [comment]
> @karsa-mistmere You want do a final vote on this one?

So far two votes for the top right one, one for bottom right, I'm also leaning more towards top right, so I guess let's go for that one. A vote bot would certainly be a nice addition. ^_^ 

Maybe we could also have a dedicated voting room on Discord? Maybe that would attract more attention than votes random issues and PRs. 😅

**Comment by ericfennis** [comment]
@danielbayley So top right it is!

**Comment by danielbayley** [comment]
> @danielbayley So top right it is!

@ericfennis Done.

**Review by ericfennis — APPROVED** [review]
Nice!

**Comment by karsa-mistmere** [comment]
@danielbayley: The circles seem not to be aligned on the grid here, especially the one in the middle, I guess it fell victim to some optimization.

**Comment by danielbayley** [comment]
> @danielbayley: The circles seem not to be aligned on the grid here, especially the one in the middle, I guess it fell victim to some optimization.

Yeah I think there is a constant battle going on here between Illustrator, Figma and @jguddas excellent, but imperfect script… Actually this icon is a really good example of the issue…

Before:
Untitled

After:
Untitled

I think Ideally we would handle optimisation in the `pre-commit` hook, and fix issues frequently introduced by various editors there, but not quite there yet…

See also https://github.com/lucide-icons/lucide/pull/1028#issuecomment-1526340123

**Comment by danielbayley** [comment]
Anyway, see latest `commit`…

File: icons/radar.svg [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="m13.4 10.6 5.7-5.7C17.3 3.1 14.8 2 12 2c-1.8 0-3.5.5-5 1.4" />
+  <path d="M2.3 9.6c-.2.8-.3 1.6-.3 2.4 0 5.5 4.5 10 10 10s10-4.5 10-10c0-1.3-.3-2.5-.7-3.6" />
+  <path d="M4 6h.01" />
+  <path d="M16.2 7.8c-2.3-2.3-6.1-2.3-8.5 0-2.3 2.3-2.3 6.1 0 8.5.2.2.3.3.5.4" />
+  <path d="M15.8 16.7c1.4-1.1 2.2-2.8 2.2-4.7" />
+  <path d="M12 18h.01" />
+  <circle cx="12" cy="12" r="2" />
```
Comment (karsa-mistmere) [inline icons/radar.svg:18]: It took me to completely redraw this from scratch and do some heavy manual optimization but I finally got it down to something quite small that's still on the grid:
```suggestion
  <path d="M19.08 4.92A9.97 9.97 0 0 0 6.98 3.35" />
  <path d="M4 6h.01" />
  <path d="M21.31 8.35A10 10 0 1 1 2.28 9.62" />
  <path d="M16.24 7.76A6 6 0 1 0 8.23 16.67" />
  <path d="M12 18h.01" />
  <path d="M17.99 11.66A6 6 0 0 1 15.77 16.67" />
  <circle cx="12" cy="12" r="2" />
  <path d="m13.41 10.59 5.66-5.66" />
```

Comment (jguddas) [reply icons/radar.svg:18]: 🤌

Comment (danielbayley) [reply icons/radar.svg]: > It took me to completely redraw this from scratch and do some heavy manual optimization but I finally got it down to something quite small that's still on the grid:

@karsa-mistmere Nice!

**Review by ericfennis — APPROVED** [review]

**Review by ericfennis — APPROVED** [review]

**Pull request merged**